acm/odh-edge/apps/telemetry/role-binding.yaml

```diff
@@ -0,0 +1,13 @@
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: otel-collector-argo
+  namespace: opendatahub-edge-collector
```

Comment:
This is needed because we are not deploying k8s native resources like with the other 2 apps?